projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-header-row.component.html

```diff
@@ -226,6 +226,7 @@
     </igx-grid-excel-style-filtering>
 </div>
 
+<!-- TODO  Remove inline styles if possible -->
 <div style="visibility: hidden; position: absolute; top: -10000px">
```

Comment:
@mddragnev you added those styles in [this commit](https://github.com/IgniteUI/igniteui-angular/commit/f97fe06c97be734c799c2b7514bff35cdfdb6db1). I want to move them to a class, but I am not sure if we need them at all, I remove them to test and I didn't see any changes.

Reply:
As long as I remember, this is used in order to generate the chips for the filtering area of the pivot grid.  I was using that div (which is hidden and out of the view)  to calculate whether we should have a dropdown button or if we have enough space for all chips. I think that as long as this div stays hidden and out of view you can move them to a class

Reply:
@desig9stein I believe this one shouldn't be made part of the theme. It can stay inline, as it's a sizing container and it doesn't really belong anywhere in the theme.